### ISSUE TITLE
Linux ?

### ISSUE THREAD

**Issue by stuaxo** [jnadro/pybgfx#1]
Any idea how to get this working on Linux ?

**Comment by jnadro** [comment]
Yup.  It would definitely need to have the linux bgfx libraries included for it to work properly.

Currently I was just manually building the bgfx libraries separately, but it would be nice for that to be automatic.

Hopefully in the future I will find some time to work on this project again and complete the bindings.
